backend/internal/integrations/meeting-finder/find_meetings_test.go

```diff
@@ -30,4 +30,12 @@ func TestFindLocalMeetings(t *testing.T) {
 	if len(meetings[0].Users) != 0 {
 		t.Error("Expected to find no users")
 	}
+
+	if meetings[0].Lat == "" {
+		t.Error("Latitude is not found")
+	}
+
+	if meetings[0].Lat == "" {
```

Comment:
Im guessing this is supposed to be Lon or Long???
```suggestion
	if meetings[0].Lon == "" {
```